calypso/service_test.go

```diff
@@ -193,38 +368,97 @@ func (s *ts) addReadAndWait(t *testing.T, write *byzcoin.Proof, Xc kyber.Point)
 }
 
 func newTS(t *testing.T, nodes int) ts {
+	return newTS2(t, nodes, 0)
+}
+
+// newTS2 initially the byzRoster and ltsRoster are the same, the extras are
+// there so that we can change the ltsRoster later to be something different.
+func newTS2(t *testing.T, nodes int, extras int) ts {
```

Comment:
this is a bad name (I introduced it), can we change it to something like `newTSWithExtras`?